core/src/main/java/com/graphhopper/routing/util/spatialrules/SpatialRuleLookup.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.util.spatialrules;
+
+import com.graphhopper.util.shapes.GHPoint;
+
+import java.util.List;
+
+/**
+ * Lookup a rule depending on the location.
+ *
+ * @author Robin Boldt
+ */
+public interface SpatialRuleLookup {
+
+    SpatialRule lookupRule(double lat, double lon);
+    SpatialRule lookupRule(GHPoint point);
+
+    void addRule(SpatialRule rule);
+
+    SpatialRule getEmptyRule();
```

Comment:
I would remove this method and make the EMPTY_RULE constant part of this interface